drivers/dma/dma_stm32_v2.c

```diff
@@ -117,7 +117,7 @@ static void (*func_ll_clear_te[])(DMA_TypeDef *DMAx) = {
 #endif /* LL_DMA_IFCR_CTEIF8 */
 };
 
-static void (*func_ll_clear_gi[])(DMA_TypeDef *DMAx) = {
+void (*func_ll_clear_gi[])(DMA_TypeDef *DMAx) = {
```

Comment:
Why did you remove the static here and declared it extern?
I understand that it is gonna used only in this file. So why?

Reply:
func_ll_clear_gi() function is used in the dma_stm32_irq_handler() of the dma_stm32.c

Reply:
OK, I missed that one! Sorry.